schema/release-schema.json

```
@@ -2135,6 +2155,41 @@
         }
       }
     },
+    "SimpleUnit": {
+      "title": "Simple Unit",
+      "description": "A simple unit consists of: an identifier for the list (scheme) from which the unit is taken, and an identifier for the category from that list being applied.",
+      "type": "object",
+      "properties": {
+        "scheme": {
+          "title": "Scheme",
+          "description": "The list from which identifiers for units of measure are taken, using the open [unitClassificationScheme](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#unit-classification-scheme) codelist. 'UNCEFACT' is recommended.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "codelist": "unitClassificationScheme.csv",
+          "openCodelist": true
+        },
+        "id": {
+          "title": "ID",
+          "description": "The unit code taken from the scheme. Although an integer is allowed, it is recommended to use a string.",
+          "type": [
+            "string",
+            "integer",
+            "null"
+          ],
```

Is there a reason to use a different description from `item/unit/id`?

Since this is a new object, we can restrict the type to string only:

```suggestion
          "description": "The identifier from the codelist referenced in the `scheme` field. Check the [unitClassificationScheme](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#unit-classification-scheme) codelist for details of how to find and use identifiers from the scheme in use.",
          "type": [
            "string",
            "null"
          ],
```